### PR TITLE
Fix insight explorer and add polis logo

### PR DESCRIPTION
**Pull request by eabz**

### CODE REVIEW

**Comment by eabz** [comment]
Working correctly now, no errors. Seems that we have to Dashify correctly the BWS and BWC.
I am using this version: https://github.com/snogcel/bitcore-wallet-service-dash
https://github.com/snogcel/bitcore-wallet-client-dash

Both branches: master_upstream

**Comment by eabz** [comment]
BWS is Dashified, but BWC must be done manual once it is installed.

File: src/pages/send/send.ts [hunk 1]
```diff
@@ -52,9 +55,11 @@ export class SendPage {
   ionViewWillEnter() {
     this.walletsBtc = this.profileProvider.getWallets({ coin: 'btc' });
     this.walletsPolis = this.profileProvider.getWallets({ coin: 'polis' });
+    this.walletsDash = this.profileProvider.getWallets({ coin: 'dash' });
     this.hasBtcWallets = !(_.isEmpty(this.walletsBtc));
-    this.hasPolisWallets = !(_.isEmpty(this.walletsPolis));
+    this.hasDashWallets = !(_.isEmpty(this.walletsDash));
```
Comment (Yoyae) [inline src/pages/send/send.ts:60]: Why replacing this.hasPolisWallets = !(_.isEmpty(this.walletsPolis)); by Dash and not add in another line ?

File: src/providers/address-book/address-book.ts [hunk 2]
```diff
@@ -23,10 +23,15 @@ export class AddressBookProvider {
     try {
       network = (this.bwcProvider.getBitcore().Address(address)).network.name;
     } catch (e) {
+    try {
       this.logger.warn('No valid bitcoin address. Trying Polis...');
       network = (this.bwcProvider.getBitcorePolis().Address(address)).network.name;
+    } catch (e) {
+      this.logger.warn('No valid bitcoin address. Trying Dash...');
+      network = (this.bwcProvider.getBitcoreDash().Address(address)).network.name;
+      return network;
```
Comment (Yoyae) [inline src/providers/address-book/address-book.ts:32]: return network; needs to be out of the catch

**Review by Yoyae — CHANGES_REQUESTED** [review]

**Review by Yoyae — APPROVED** [review]

**Pull request merged**